src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -405,6 +511,7 @@ export class QuickBar extends LitElement {
 
         mwc-list-item {
           width: 100%;
+          text-transform: capitalize;
```

Comment:
Do we want capitalize every word or just the first?
```css
mwc-list-item::first-letter {
    text-transform: capitalize;
}
```

Reply:
My intention was every word. The only issue there is with "Navigate To ____" since the word "to" isn't normally capitalized.

What if we instead did it like VS Code:

* Navigate: Overview
* Navigate: Profile
* Reload: Input Boolean
* Reload: Automations
* Server: Restart
* Etc...

Reply:
I'm cool with that

Reply:
I think I'm going to leave that change out of this PR. It sounds like something that might mess with translations, and may be a better thing to wait on until we implement command "category" styling.